README.md

```diff
@@ -28,6 +28,12 @@ Finally, user access, discovery, and analysis is facilitated by a ReactJS web ap
 a RESTful API, or a web socket API for streaming data feeds.
 
 ## local development
+### local environment setup
+* [Setup needed to prepare workstation](https://github.com/Datastillery/smartcitiesdata/wiki/Setup)
+* [Windows](https://github.com/Datastillery/smartcitiesdata/wiki/Windows-Setup)
+* [macOS](https://github.com/Datastillery/smartcitiesdata/wiki/macOS-Setup)
+* [Linux](https://github.com/Datastillery/smartcitiesdata/wiki/Linux-Setup)
+
```

Comment:
Can we also update the content in these links. The directions only tell you how to install the software you'll need for the platform they don't actually give you directions on how to setup the environment.